packages/elements/src/components/ino-table/readme.md

```diff
@@ -5,21 +5,29 @@ The component is based on the mdc-data-table.
 
 ## Usage
 
-The component is composed of multiple sub-components and can be used as follows:
+The component comprises two sub-components: `ìno-table-row` and `ino-table-cell`. The `ino-table-row` can be used to
+add a row to the table. To add a header row, all you have to do is to add the `ino-header-row` property to the component
+and set `slot=header`.
+Correspondingly, the `ino-table-cell` can be used to add a column to the table. You should always set the `ino-numeric` 
```

Comment:
> the `ino-table-cell` can be used to add a **column** to the table

Is **column** correct here or do you mean cell?

Reply:
I actually meant column since a cell in a row represents a column.

Reply:
Still, I see what you mean. A single column consists of multiple cells. Adding a single cell to a row does not necessarily mean that you're adding an entire row.

Reply:
I just changed it. Take a look at it now and let me know if you think it's better this way or if it is still unclear.